src/api/roles.js

```diff
@@ -0,0 +1,8 @@
+import { ROLES } from 'test/mocks/roles';
+
+// eslint-disable-next-line
+export const getApiRoles = () => (
```

Comment:
always specify which rule is being disabled

Reply:
OK fixed